scripts/image/create-image-no-backports.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#
+# Simple script to use sdm with plugins
+# Edit the text inside the EOF/EOF as appropriate for your configuration
+# ** Suggestion: Copy this file to somewhere in your path and edit your copy
+#    (~/bin is a good location)
+#
+
+function errexit() {
+    echo -e "$1"
+    exit 1
+}
+
+[ $EUID -eq 0 ] && sudo="" || sudo="sudo"
+
+IMAGE="2024-11-19-raspios-bookworm-arm64-lite.img"
+
+# service dbus start
+
+if ! test -f $IMAGE; then
+  curl -O https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-11-19/2024-11-19-raspios-bookworm-arm64-lite.img.xz
+  xz -d $IMAGE.xz
+fi
+
+[ "$(type -t sdm)" == "" ] && errexit "? sdm is not installed"
+
+assets="."
+
+[ -f $assets/my.plugins ] &&  mv $assets/my.plugins $assets/my.plugins.1
+
+(cat <<EOF
+# Plugin List generated $(date +"%Y-%m-%d %H:%M:%S")
+
+# Delete user pi if it exists
+# https://github.com/gitbls/sdm/blob/master/Docs/Plugins.md#user
+user:deluser=pi
+
+# Add a new user
+# https://github.com/gitbls/sdm/blob/master/Docs/Plugins.md#user
+user:adduser=ladmin|password=Numark234|addgroup=audio
+
+# Install btwifiset (Control Pi's WiFi from your phone)
+# https://github.com/gitbls/sdm/blob/master/Docs/Plugins.md#btwifiset
+# btwifiset:country=GB|timeout=30
+
+# Install apps
+# https://github.com/gitbls/sdm/blob/master/Docs/Plugins.md#apps
+apps:name=pipewire-jack|apps=pipewire-jack
+apps:name=ui-development|apps=python3-brlapi raspberrypi-ui-mods libjson-glib-dev libgtk-4-dev 
+apps:name=libusb-dev|apps=libusb-dev libsystemd-dev
+apps:name=autopoint-gettext|apps=autopoint gettext
+apps:name=libasound2-dev|apps=libasound2-dev libjack-dev libsamplerate0-dev libsndfile1-dev
+apps:name=python3-dev|apps=python3-dev libcairo2-dev
+apps:name=git-automake-libtool|apps=git automake libtool
+apps:name=overwitch-dependencies|apps=libusb-1.0-0-dev,qpwgraph
+
+# Configure network
+# https://github.com/gitbls/sdm/blob/master/Docs/Plugins.md#network
+# network:ctype=wifi|wifi-ssid=ONTOLEDGY_WIFI|wifi-password=PAKI4200|wificountry=GB|autoconnect=true
```

Comment:
Did you mean to commit your wifi pass? It might be worth putting that and the SSID in an env var. To prevent it from being as much of a hassle lately I've been letting raspi-config do the network stuff and excluding it from the SDM scripts but I'm open to whatever you think is best.